charts/ingress-azure/templates/deployment.yaml

```diff
@@ -27,6 +27,9 @@ spec:
         checksum/config: {{ print .Values | sha256sum }}
         prometheus.io/scrape: "true"
         prometheus.io/port: {{ .Values.kubernetes.httpServicePort | quote}}
+        {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
```

Comment:
you are not iterating over them here as everywhere else...this will not work as intended